sherpa/astro/xspec/__init__.py

```diff
@@ -103,7 +103,10 @@
 
 from __future__ import annotations
 
+from collections.abc import Callable
 from contextlib import suppress
+from collections.abc import Callable
```

Comment:
```suggestion
```

Unless GH is lying to me about line 106 ....

Reply:
That's an error from the rebase. Sorry.